CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.1.15 (unreleased)
+- Fix error of ignoring restore_point field in resource_vkcs_db_instance and resource_vkcs_db_cluster
```

Comment:
s/restore_point field/restorePoint argument/ - because of this is info for users, but they know restoryPoint argument only